content/applications/productivity/mail_plugins/outlook.rst

```diff
@@ -75,6 +75,25 @@ Connect your database
       Only a limited amount of *Company Insights* (*Lead Enrichment*) requests are available as a
       trial. This feature requires :ref:`prepaid credits <mail_plugins/pricing>`.
 
+   .. warning::
+      If, after a short while, the panel is still empty. It is possible that your browser cookies
```

Comment:
I made some small changes to this sentence to improve sentence flow.

`If, after a short while, the panel is still empty, it is possible that your browser cookie`